ovve_ui/utils/params.py

```diff
@@ -5,80 +5,198 @@
 from typing import Union
 
 
-# Params defined in serialpacketv0.26.pptx
 class Params():
+    """
+    Params defined in serialpacketv0.26.pptx
+    """
     def __init__(self) -> None:
-        self.run_state: int = 0
-        self.seq_num: int = 0
-        self.packet_version: int = 0
-        self.mode: int = 0
-        self.resp_rate_meas: int = 0
-        self.resp_rate_set: int = 0
-        self.tv_meas: int = 0
-        self.tv_set: int = 0
-        self.ie_ratio_meas: int = 0
-        self.ie_ratio_set: int = 0
-        self.peep: int = 0
-        self.ppeak: int = 0
-        self.pplat: Union[int, float] = 0
-        self.pressure: Union[int, float] = 0
-        self.flow: Union[int, float] = 0
-        self.tv_insp: int = 0
-        self.tv_exp: int = 0
-        self.tv_min: int = 0 
-        self.control_state: int = 0
-        self.battery_level: int = 0
+        self._param: dict = {
+            "seq_num": 0,
```

Comment:
We're missing run_state here.  That was a late addition during the call today.  You might have started on this before I made that change.

Reply:
Cool! I'll add `run_state` as a parameter within the `dict` and as a property :-)